### DISCUSSION TITLE
Libre 2 - OPPAlgorithm2 without calibrations

### DISCUSSION THREAD

**Discussion by Thomas-Wolff** [NightscoutFoundation/xDrip#1963] [Q&A]
Hello, I have read about this possibility.
I have already installed the new OOP2 build and am currently running it with the nightly xDrip build from 28 January 2022.

Unfortunately, I still have to calibrate it, but the data that came in after a first initial calibration was so accurate that I could do without further calibrations. 

What are the plans for xDrip so that the Libre2 does not need any more calibrations in the future? @Tzachi-dar has already put a corresponding build on the net, is it advisable to install it or wait and see.

It would help me if I could get some feedback on the current status or future plans.

With kind regards Thomas

**Comment by psonnera** [comment]
@Thomas-Wolff this should be now available with https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.05.02 did you check it and allow this question to be answered?
Note: make sure you use a recent nightly and version of OOP2 as there has been a bugfix https://xdrip.readthedocs.io/en/latest/use/OOP/#oop2
